### ISSUE TITLE
yaml-language-server 0.6.0 start failed

### ISSUE THREAD

**Issue by pytimer** [redhat-developer/vscode-yaml#249]
Hi, i install the latest version 0.6.0 in mac 10.14.5, i found the error in the vscode. I'm not sure this error should report this repo or https://github.com/redhat-developer/yaml-language-server.

The error logs below:

```txt
/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:199
        const fpa = new jsonSchemaService_1.FilePatternAssociation(globPath);
                    ^

TypeError: jsonSchemaService_1.FilePatternAssociation is not a constructor
    at isKubernetes (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:199:21)
    at validateTextDocument (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:228:62)
    at Timeout._onTimeout (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:217:9)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
[Info  - 5:25:46 PM] Connection to server got closed. Server will restart.
/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:199
        const fpa = new jsonSchemaService_1.FilePatternAssociation(globPath);
                    ^

TypeError: jsonSchemaService_1.FilePatternAssociation is not a constructor
    at isKubernetes (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:199:21)
    at validateTextDocument (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:228:62)
    at Timeout._onTimeout (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:217:9)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
[Info  - 5:25:46 PM] Connection to server got closed. Server will restart.
/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:199
        const fpa = new jsonSchemaService_1.FilePatternAssociation(globPath);
                    ^

TypeError: jsonSchemaService_1.FilePatternAssociation is not a constructor
    at isKubernetes (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:199:21)
    at validateTextDocument (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:228:62)
    at Timeout._onTimeout (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:217:9)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
[Info  - 5:25:47 PM] Connection to server got closed. Server will restart.
/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:199
        const fpa = new jsonSchemaService_1.FilePatternAssociation(globPath);
                    ^

TypeError: jsonSchemaService_1.FilePatternAssociation is not a constructor
    at isKubernetes (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:199:21)
    at validateTextDocument (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:228:62)
    at Timeout._onTimeout (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:217:9)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
[Info  - 5:25:47 PM] Connection to server got closed. Server will restart.
/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:199
        const fpa = new jsonSchemaService_1.FilePatternAssociation(globPath);
                    ^

TypeError: jsonSchemaService_1.FilePatternAssociation is not a constructor
    at isKubernetes (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:199:21)
    at validateTextDocument (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:228:62)
    at Timeout._onTimeout (/Users/pytimer/.vscode/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:217:9)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
[Error - 5:25:48 PM] Connection to server got closed. Server will not be restarted.
```

**Comment by pytimer** [comment]
This issue i will close, because i found https://github.com/redhat-developer/yaml-language-server/issues/202 the same as my question.

**Comment by JPinkney** [comment]
@pytimer This should be fixed in 0.6.1
